src/Symfony/Component/CssSelector/README.md

```diff
@@ -36,12 +36,15 @@ names are lower-cased, the following extra pseudo-classes are supported:
 Resources
 ---------
 
+  * [Documentation](https://symfony.com/doc/current/components/css_selector.html)
+  * [Contributing](https://symfony.com/doc/current/contributing/index.html)
+  * [Report issues](https://github.com/symfony/symfony/issues) and
+    [send Pull Requests](https://github.com/symfony/symfony/pulls)
+    in the [main Symfony repository](https://github.com/symfony/symfony)
+
+Third Party Dependencies
```

Comment:
this is not really a dependency. The next paragraph is explaining where the implementation comes from